### PR TITLE
Delete entity &nbsp;

### PR DESCRIPTION
**Pull request by fonziset**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [ ] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-12345

### CODE REVIEW

**Comment by Crizz0** [comment]
Hi,

please name the PR according to the Ticket, which are you fixing. Take care that you fillout the PR-template correctly and that your commit message follows the format.

Regards

File: phpBB/adm/style/overall_header.html [hunk 1]
```diff
@@ -118,7 +118,7 @@
 		<div id="acp">
 				<div id="content">
 					<div id="menu">
-						<p>{L_LOGGED_IN_AS}<br /><strong>{USERNAME}</strong> [&nbsp;<a href="{U_LOGOUT}">{L_LOGOUT}</a>&nbsp;][&nbsp;<a href="{U_ADM_LOGOUT}">{L_ADM_LOGOUT}</a>&nbsp;]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
+						<p>{L_LOGGED_IN_AS} <strong>{USERNAME}</strong> [ <a href="{U_LOGOUT}">{L_LOGOUT}</a> ] [ <a href="{U_ADM_LOGOUT}">{L_ADM_LOGOUT}</a> ]</p>
```
Comment (Crizz0) [inline phpBB/adm/style/overall_header.html]: Why did you also remove the br-tag?

Comment (Skouat) [inline phpBB/adm/style/overall_header.html]: Hi,

Removing the br-tag and the nbsp between brackets `[ ]` could cause a display as bellow
```
You are logged in as: user1 [
 Logout ][ ACP Logout
 ]
```
IMO, only `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;` should be removed.

Comment (hanakin) [reply phpBB/adm/style/overall_header.html]: not true pls always remove all br tags! even if it requires classes added to wrappers!

**Comment by hanakin** [comment]
pls create a bug ticket for this issue and properly submit a new pr for this! https://area51.phpbb.com/docs/dev/31x/development/git.html

**Comment by CHItA** [comment]
Closing this as per the comment of @hanakin. Please resend the PR following his notes. Thank you for contributing to the project.

**Pull request closed without merging**